invenio_rdm_records/resources/serializers/datacite/schema.py

```diff
@@ -192,6 +193,13 @@ def _read_title_type(self, id_):
             system_identity
         )._record
 
+    def _read_description_type(self, id_):
+        """Retrieve description type record using service."""
+        return vocabulary_service.read(
+            ('descriptiontypes', id_),
+            system_identity
+        )._record
```

Comment:
Recalling the standup a few days ago: We are having this as a pattern all around, at resource level we should not access `_record` (data layer). Is there any particular reason for this? can we just use the `ResultItem` coming from the service level?

The `.get("title")` used below should still work since it inherits `dict`

Reply:
Nicely spotted, thanks!